src/Collection/Collection.router.spec.ts

```diff
@@ -1374,6 +1374,78 @@ describe('Collection router', () => {
         })
       })
 
+      describe('and interating with the database fails', () => {
+        let itemCurationDeleteSpy: jest.SpyInstance<
+          ReturnType<typeof ItemCuration.delete>
+        >
+
+        beforeEach(() => {
+          ;(Item.findByIds as jest.Mock).mockResolvedValueOnce([dbItemMock])
+          ;(ItemCuration.findLastCreatedByCollectionIdAndStatus as jest.Mock).mockResolvedValueOnce(
+            undefined
+          )
+          jest.spyOn(ethers.utils, 'verifyMessage').mockReturnValue('0x')
+
+          jest.spyOn(SlotUsageCheque, 'create').mockResolvedValueOnce({})
```

Comment:
Would you mind adding here an id so we can later assert that the delete procedure was called with this id?

Reply:
The same goes for the `delete` method of the `SlotUsageCheque`.